lib/annotator-custom-editor.js

```diff
@@ -1,46 +1,49 @@
-// Generated by CoffeeScript 1.9.1
-(function() {
-  var CustomEditor,
-    extend = function(child, parent) { for (var key in parent) { if (hasProp.call(parent, key)) child[key] = parent[key]; } function ctor() { this.constructor = child; } ctor.prototype = parent.prototype; child.prototype = new ctor(); child.__super__ = parent.prototype; return child; },
-    hasProp = {}.hasOwnProperty;
+'use strict';
 
-  Annotator.Plugin.CustomEditor = CustomEditor = (function(superClass) {
-    extend(CustomEditor, superClass);
+var _classCallCheck = function (instance, Constructor) { if (!(instance instanceof Constructor)) { throw new TypeError('Cannot call a class as a function'); } };
 
-    function CustomEditor() {
-      return CustomEditor.__super__.constructor.apply(this, arguments);
+var _createClass = (function () { function defineProperties(target, props) { for (var i = 0; i < props.length; i++) { var descriptor = props[i]; descriptor.enumerable = descriptor.enumerable || false; descriptor.configurable = true; if ('value' in descriptor) descriptor.writable = true; Object.defineProperty(target, descriptor.key, descriptor); } } return function (Constructor, protoProps, staticProps) { if (protoProps) defineProperties(Constructor.prototype, protoProps); if (staticProps) defineProperties(Constructor, staticProps); return Constructor; }; })();
```

Comment:
If moving to ES6, I suggest using `let` instead of `var` (or `const` where appropriate)